clients/ordering/package.json

```diff
@@ -4,13 +4,18 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "3.9.2",
+    "@material-ui/icons": "3.0.2",
     "@types/jest": "24.0.0",
     "@types/node": "10.12.21",
     "@types/react": "16.8.2",
     "@types/react-dom": "16.8.0",
+    "@types/react-router": "^4.4.4",
+    "@types/react-router-dom": "^4.3.1",
+    "ajv": "^6.8.1",
```

Comment:
What's `ajv` for?

Reply:
Not sure how this got in here. I can revmove it.